lib/semian/atomic_integer.rb

```diff
@@ -0,0 +1,28 @@
+module Semian
+  class AtomicInteger #:nodoc:
+    attr_accessor :value
+
+    def initialize(_name, _permissions)
```

Comment:
Why the underscored arguments here? These don't matter for `AtomicInteger`. They only matter for the SysV adapter which will override the initializer regardless.